controllers/notifyController.php

```diff
@@ -395,14 +425,14 @@ private function updateBdInfo()
         global $wpdb;
 
         $this->checkDbTable();
-        $tableName = $wpdb->prefix.PG_CART_PROCESS_TABLE;
+        $tableName = $wpdb->prefix . PG_OLD_CART_PROCESS_TABLE;
```

Comment:
If it's a new installation the tables won't exist.

Reply:
fixed